enhancements/csvless-bundles.md

```diff
@@ -24,209 +24,488 @@ status: implementable
+  - displayName: Secret Name
+    description: "If Kiali is configured with auth.strategy 'login', an admin must create a Secret with credentials ('username' and 'passphrase') which will be used to authenticate users logging into Kiali. This setting defines the name of that secret. Default: kiali"
+    path: spec.deployment.secret_name
+    capabilities:
+    - 'urn:alm:descriptor:com.tectonic.ui:selector:core:v1:Secret'
+  - displayName: Verbose Mode
+    description: "Determines the priority levels of log messages Kiali will output. Typical values are '3' for INFO and higher priority messages, '4' for DEBUG and higher priority messages (this makes the logs more noisy). Default: 3"
+    path: spec.deployment.verbose_mode
+    capabilities:
+    - 'urn:alm:descriptor:com.tectonic.ui:label'
+  - displayName: View Only Mode
+    description: "When true, Kiali will be in 'view only' mode, allowing the user to view and retrieve management and monitoring data for the service mesh, but not allow the user to modify the service mesh. Default: false"
+    path: spec.deployment.view_only_mode
+    capabilities:
+    - 'urn:alm:descriptor:com.tectonic.ui:booleanSwitch'
+  - displayName: Web Root
+    description: "Defines the root context path for the Kiali console, API endpoints and readiness/liveness probes. Default: / (when deployed on OpenShift; /kiali (when deployed on Kubernetes)"
+    path: spec.server.web_root
+    capabilities:
+    - 'urn:alm:descriptor:com.tectonic.ui:label'
+- apiVersion: monitoring.kiali.io/v1alpha1
+  kind: MonitoringDashboard
+  description: A configuration file for defining an individual metric dashboard.
+  displayName: Monitoring Dashboard
   descriptors:
-  - description: The name of the CatalogSource that owns this package
-    displayName: Package Manifest
-    path: status.catalogSource
+  - displayName: Title
+    description: "The title of the dashboard."
+    path: title
     capabilities:
-    - urn:alm:descriptor:text
+    - 'urn:alm:descriptor:com.tectonic.ui:label'
 ```
 
-The `apis` element is a special case and is present to enable to organization of [descriptors](https://github.com/operator-framework/operator-lifecycle-manager#descriptors). It will be kept up to date with `provided APIs` inferred by the `opm bundle build` command upon execution.
+_**Note:** `capabilities` is the equivalent of the `x-discriptors` field found in CSVs._
 
-_**Note:** The `descriptors.path` element's value is qualified with the `status` subresource. For comparision, a CSV contains two types of descriptors -- one for spec, the other for status -- both with path elements that don't contain this additional qualifier._
+The `annotations.yaml` file contains the set of labels to be projected onto the container image representation of the bundle. More importantly, it contains the top-level discriminator field used to identify the mediatype of the bundle on-disk: `operators.operatorframework.io.bundle.mediatype: registry+v2`.
```

Comment:
How do the annotations that represent paths to manifest and metadata subdirectories interact with v2 bundles? Do we need to update the documentation of those annotations to indicate that they only have meaning when mediatype is v1?

Reply:
> How do the annotations that represent paths to manifest and metadata subdirectories interact with v2 bundles?

They don't, it's purely based on directory structure.

> Do we need to update the documentation of those annotations to indicate that they only have meaning when mediatype is v1?

It's probably something to point out in the spec for `registry+v2`, yes.